### PR TITLE
Update bintray deploy information about deb part

### PR DESCRIPTION
**Pull request by camlafit**
Some lacking information about deb repository.

* upload pattern must respect automatic layout, some informations are mandatories, 
* and if your file are on root script, relative path must be set as "includePattern": "./(.*\\.deb)",

### CODE REVIEW

File: user/deployment/bintray.md [hunk 1]
```diff
@@ -114,11 +114,14 @@ The descriptor is in JSON file format in three sections:
 
 #### Package Section
 
-Bintray package information. In case the package already exists on Bintray, only the name, repo and subject fields are mandatory.
+Bintray package information. Following information are mandatories on open source project.
```
Comment (BanzaiMan) [inline user/deployment/bintray.md]: → "mandatory on open source projects."

File: user/deployment/bintray.md [hunk 2]
```diff
@@ -114,11 +114,14 @@ The descriptor is in JSON file format in three sections:
...
 
 - `name` is the Bintray package name
 - `repo` is the Bintray repository name
 - `subject` is the Bintray subject, which is either a user or an organization
+- `vcs_url` is the Bintray vcs url, could be a github repository url or either vcs repository url
+- `licenses` is the [Bintray licences](https://bintray.com/docs/api/#url_licenses), which is either a list with at list on item.
```
Comment (BanzaiMan) [inline user/deployment/bintray.md]: Could you elaborate on "which is either a list with at list on item."? Did you mean "which is either a list or an item"? Or "which is a list with at least one item"? Or something else entirely?

Comment (camlafit) [inline user/deployment/bintray.md]: it's a list with at lest one item,
Congratulation to have understood my bad english :)

File: user/deployment/bintray.md (hunk 1 above)
Comment (camlafit) [inline user/deployment/bintray.md]: corrected thanks

I set information at plural, i'm not sure about this

Comment (BanzaiMan) [inline user/deployment/bintray.md]: I missed that. "Information" is singular. So, "Following information is…".

**Comment by plaindocs** [comment]
Thanks!

**Pull request merged**